### ISSUE TITLE
CSS / AMP: Bug with minification an amp-boilerplate

### ISSUE THREAD

**Issue by pperzyna** [tdewolff/minify#300]
I believe I am using your library via Hugo. I found following error during a minification a CSS.

Un-minified:
```css
body {
	-webkit-animation: -amp-start 8s steps(1,end) 0s 1 normal both;
	-moz-animation: -amp-start 8s steps(1,end) 0s 1 normal both;
	-ms-animation: -amp-start 8s steps(1,end) 0s 1 normal both;
	animation: -amp-start 8s steps(1,end) 0s 1 normal both
}
```

Minified:

```css
body{-webkit-animation:-amp-start 8s steps(1,end)0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end)0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end)0s 1 normal both;animation:-amp-start 8s steps(1,end)0s 1 normal both}
```

and the issue is here `steps(1,end)0s`

**Comment by tdewolff** [comment]
Hi pperzyna, thanks for raising the issue. The removed space between `)` and `0` is actually intentional. I've ran an extra test to be sure it works and for me in Chrome it's working as intended. Could you indicate which browser and version is giving you problems?

**Comment by pperzyna** [comment]
Sure, you can see example on my website [pperzyna.com](https://pperzyna.com/amp/blog/how-to-move-metrics-from-locust_io-to-grafana-via-prometheus/). This is an AMP version minify by Hugo (with you lib), and the source code is here: [pperzyna.github.io](https://github.com/pperzyna/pperzyna.github.io/blob/master/themes/pperzyna/layouts/partials/site-amp.html#L7). 

It doesn't pass [Google AMP Validation](https://search.google.com/test/amp?utm_source=gws&utm_medium=onebox&utm_campaign=suit&url=https%3A%2F%2Fpperzyna.com%2Famp%2Fblog%2Fhow-to-move-metrics-from-locust_io-to-grafana-via-prometheus%2F), because of that.

**Comment by tdewolff** [comment]
This should've been handled by https://github.com/tdewolff/minify/commit/3b9d2321f0e915b8771d7ed4415970e3944796d3. Any content in `<style amp-boilerplate>` should not be touched. Whitespace should also be reduced as it is treated has plain text in the minifier.

Looks like Hugo is running an old version of minify which doesn't have that fix (which appeared in v2.6.2).
